lib/dogma/formatter/emacs.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Dogma.Formatter.Emacs do
```

Comment:
Great :)

I'd like to go against `machine` since I want to support other easily machine readable formats.